base_mail_bcc/models/ir_mail_server.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Thomas Rehn (initOS GmbH)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
+from email.Utils import COMMASPACE
+
+
+class IrMailServer(models.Model):
+    _inherit = "ir.mail_server"
+
+    @api.model
+    def send_email(self, message, **kwargs):
```

Comment:
say `*args, **kwargs` here, because otherwise, it will break if some code calls this function with positional arguments. Better is to exactly repeat the parameters as defined by super